apps/weather/clkinfo.js

```diff
@@ -8,9 +7,9 @@
       if(weather){
         weather.temp = require("locale").temp(weather.temp-273.15);
         weather.feels = require("locale").temp(weather.feels-273.15);
```

Comment:
My watch says "Feels: -262 C". I suspect it's because we convert to celcius both here and on Gadgetbridge side?

Gadgetbridge:
<img width="1020" height="22" alt="image" src="https://github.com/user-attachments/assets/17cb33d9-f325-432c-940b-a0ab456e1a3c" />

But if that's the case I'm not sure why the same doesn't happen with the current temp value. 🤔

Reply:
Found the bug at https://github.com/espruino/BangleApps/pull/3910#discussion_r2280578552